shared/src/commonMain/kotlin/com/intive/picover/shared/main/navigation/view/MainScreen.kt

```diff
@@ -79,7 +81,9 @@ private fun UserAuthorizedContent(snackbarHostState: SnackbarHostState) {
 							TabNavigationRailItem(ProfileTab)
 						}
 					}
-					CurrentTab()
+					Column(Modifier.padding(paddingValues)) {
```

Comment:
Fixes visibility of FAB buttons on Parties and Photos screen

Reply:
I created an issue for that #169, thank you for reminder

Reply:
But besides the FAB, nested screen was aligned to bottom of bottom navigation bar before that change